servers/jax-rs-tests/src/main/java/org/projectnessie/jaxrs/tests/AbstractRelativeReferences.java

```diff
@@ -803,11 +1515,16 @@ void getDiffToRefAmbiguousDetached() {
     checkError(error, BAD_REQUEST, "To hash must contain a starting commit ID.");
```

Comment:
Would it make sense to add quotation marks around `To`, i.e. `"To" hash ...`? In current form the whole message is not very readable, IMHO.

This is not really related to the purpose of this PR, but we're changing how validation messages are constructed :thinking:

Reply:
I used `"To" hash` as suggested... if that's not enough, we could maybe use `Hash to start the diff from ...` wdyt?